Directory.Build.targets

```diff
@@ -77,7 +77,7 @@
           Condition="'@(NETStandardCompatError)' != ''"
           Inputs="%(NETStandardCompatError.Identity)"
           Outputs="unused"
-          BeforeTargets="GetFiles">
+          BeforeTargets="_GetPackageFiles;GetFiles">
```

Comment:
Why is this change necessary? Wasn't this working on csproj before?

Reply:
No it wasn't working on the csproj, the extensions projects like Microsoft.Extensions.Primitives were missing this file.
GetFiles is a custom target defined by our old pkgproj system and is not used by dotnet pack.

Reply:
I see. Can we avoid the custom target here and instead just use `Pack`?